gandalf.js

```diff
@@ -1,51 +1,57 @@
-/**
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.